front/src/applications/operationalStudies/hooks/useSimulationResults.ts

```diff
@@ -128,7 +128,7 @@ const useSimulationResults = (infraId: number): SimulationResults | undefined =>
     !rawPathProperties ||
     !rollingStock
   ) {
-    return undefined;
+    return { isValid: false, train: train ?? undefined, rollingStock: rollingStock ?? undefined };
```

Comment:
You can simplify this
```suggestion
    return { isValid: false, train, rollingStock };
```

Reply:
I dropped the commit ;)